app/shared/shared/mapping/models.py

```diff
@@ -50,6 +50,36 @@ class MappingStatus(models.Model):
     display_name = models.CharField(max_length=64)
 
 
+class JobStage(models.Model):
```

Comment:
On retrospect I think we should actually do a new django "app" for the Jobs code, as we slice by domain here.

So similar to `app/shared/shared/files` - do a new `app/shared/shared/jobs` - with these models, migrations, and initialise the app in django `settings.py`